src/components/IdentityManager/IdentityManager.js

```diff
@@ -6,6 +6,7 @@ const identityEventTypes = {
   CLEAR: 'CLEAR',
```

Comment:
Let's create an issue to remove `CLEAR` and its accompanying flow methods and props in `aragon`, `aragon.js` and `aragon-apps`

Reply:
Is remove is completely replacing clear?

Reply:
Yep, you can `clear` by `removing all`

Reply:
Follow up: https://github.com/aragon/aragon/issues/838